packages/migrations/app/phpunit.xml

```diff
@@ -1,21 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This configuration was never used and in other packages we do not provide a custom configuration, so I've removed it.